src/main/java/com/parasoft/demoapp/config/activemq/RefreshableMessageListener.java

```diff
@@ -34,6 +53,22 @@ private void registerListener(String destinationName) {
         jmsListenerEndpoint.setId(ID_PREFIX + destinationName);
         jmsListenerEndpointRegistry.
                 registerListenerContainer(jmsListenerEndpoint, jmsQueueListenerContainerFactory, true);
+        listenedListenerContainers.put(destinationName, jmsListenerEndpointRegistry.getListenerContainer("id:" + destinationName));
+    }
+
+    public void refreshDestination(String destinationName) throws GlobalPreferencesNotFoundException, GlobalPreferencesMoreThanOneException {
```

Comment:
Remove the GlobalPreferencesNotFoundException, GlobalPreferencesMoreThanOneException since there is not exception to throw.